source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -714,16 +725,11 @@ class NiDAQmxDriverApiTests : public Test {
     EXPECT_EQ(::grpc::Status::OK.error_code(), status.error_code());
   }
 
-  void EXPECT_DAQ_ERROR(int32_t expected_error, const std::string& error_message)
+  void EXPECT_DAQ_ERROR(int32_t expected_error, const nidevice_grpc::experimental::client::grpc_driver_error& ex)
```

Comment:
This seems applicable to all of the drivers. Have you considered single-sourcing it in a header file?

Reply:
Added a new header. That should help tremendously.